lib/ansible/module_utils/common/collections.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c), Sviatoslav Sydorenko <ssydoren@redhat.com> 2018
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+"""Collection of low-level utility functions."""
```

Comment:
Add to this docstring something like
> This module is intended only for internal use.  It will go away once the bundled copy of six includes equivalent functionality.  Third parties should not use this.

Reply:
I believe you meant `_collections_compat`, not `collections`

Reply:
Oops, yes.  That's where I meant it to be